### PR TITLE
Fix MangaFast

### LINKED ISSUE #8277: MangaFast: Missing chapter names, images and search results
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

TachiyomiJ2K 1.2.1

### Android version

Android 11

### Source information

MangaFast 1.2.12

### Steps to reproduce

1. Open MangaFast extension
2. Try to navigate it or open a Manga

### Expected behavior

It should load all the content available, chapters, summary and cover. 

### Actual behavior

Instead it is a blank, nothing seems to load (see images).

Screenshot_20210726-000919__01
Screenshot_20210726-002213

### Other details

WebView "sort of" works; the website seems to have some pretty aggressive redirecting ads that I believe are getting in the way.

### PR DESCRIPTION
**Pull request by FourTOne5**
Closes #8277

### CODE REVIEW

**Comment by FourTOne5** [comment]
Should I use popular from Front-page? https://mangafast.net/list-manga isn't actually popular.

**Comment by druppelt** [comment]
You could zip the sections "Trending Manhua",  "Trending Manhwa", "Top Weekly Comic", " Top Full-Color Comic", then concat "Full Color Manhwa & Manhua" and remove duplicates, but I'm not sure if it's worth the effort, especially regarding maintainability.

**Comment by FourTOne5** [comment]
> Also I'd rather have the fix merged asap and change the popular list later on, if at all. It has been /list-manga for quite some time and afaik no one has complained yet :)

/list-manga was popular until this site change it now lists all the manga a-z

**Comment by aditvalue** [comment]
Can you tell me when this will be available in-app?

**Comment by FourTOne5** [comment]
Most of the things are done. Only the popular needs to be changed according
to the above comment and some code error needs to addressed. I personally
have no time atm(The latest commit was done from my phone xD).

**Comment by Riztard** [comment]
i think better use /list-manga instead of only few popular manga from the front page

**Comment by FourTOne5** [comment]
GTG I guess.

File: src/en/mangafast/src/eu/kanade/tachiyomi/extension/en/mangafast/MangaFast.kt [hunk 1]
```diff
@@ -1,17 +1,24 @@
 package eu.kanade.tachiyomi.extension.en.mangafast
 
 import eu.kanade.tachiyomi.network.GET
-import eu.kanade.tachiyomi.source.model.FilterList
-import eu.kanade.tachiyomi.source.model.Page
-import eu.kanade.tachiyomi.source.model.SChapter
-import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.source.model.*
```
Comment (Riztard) [inline src/en/mangafast/src/eu/kanade/tachiyomi/extension/en/mangafast/MangaFast.kt:5]: dont use wildcard, it will cause failed to build

Comment (FourTOne5) [reply src/en/mangafast/src/eu/kanade/tachiyomi/extension/en/mangafast/MangaFast.kt:5]: I see

**Pull request closed without merging**